ssc/cmod_cashloan.cpp

```diff
@@ -990,7 +1104,7 @@ class cm_cashloan : public compute_module
 			lcoe_real /= npv_energy_real;
 		}
 
-		double npv_energy_nom = npv( CF_energy_net, nyears, nom_discount_rate );
+		double npv_energy_nom = npv( CF_energy_sales, nyears, nom_discount_rate ) + npv(CF_parasitic_cost, nyears, nom_discount_rate);
```

Comment:
I thought CF_parasitic_cost went in the numerator. This is the energy term in the denominator of the nominal LCOE.